znake/activate

```diff
@@ -1,9 +1,18 @@
 # Source this file to create and activate a Znake virtual environment
-python3.6 -m venv .venv
+python3.8 -m venv .venv
```

Comment:
I think I made a working verision of Znake and friends available here ages ago:
https://github.com/ZenDevelopmentEcosystem

I'm available if you want to make an attempt at resurrecting this stuff.

Reply:
I was thinking of forking this repo a bunch of times and then move files around in each repo, to maintain the history and origin ^^

Reply:
I have an ambition of resurrecting at least some of these projects :D Looking at the code, I can see that a lot of love and effort has been put into this code, and, frankly, I haven't found anything remotely as advanced and yet simple to use, anywhere.

Reply:
All new development in these products are done at https://github.com/ZenDevelopmentEcosystem.
I hope to have some more time to work soon this, especially k2 and make it use-able in non-Zenterio context.
Any help is greatly appriciated. Feel free to reach out and discuss future roadmap.

Reply:
What would be the best forum for discussion? Feels a bit clunky in a review.
Maybe create an issue for each project to be imported? I noticed that neither zaf nor k2-core have any actual code in them